pkg/apis/configuration/v1/configsource.go

```diff
@@ -7,13 +7,33 @@ type ConfigSource struct {
 	SecretValue SecretValueFromSource `json:"secretKeyRef,omitempty"`
 }
 
+// ConfigPatch is a JSON patch to add values from secrets in the same namespace with KongPlugin
+// to the generated configuration of plugin in Kong.
```

Comment:
```suggestion
// ConfigPatch is a JSON patch (RFC6902) to add values from Secret to the generated configuration.
// It is an equivalent of the following patch: 
// `{"op": "add", "path": {.Path}, "value": {.ComputedValueFrom}}`.
```
I intentionally omit being specific to Plugins because it may be a reusable piece.

Reply:
Done.